codemeta.json

```diff
@@ -369,6 +369,6 @@
         "click>=7.0",
         "pydantic>=2.5.0",
         "iminuit>=2.8.0",
-        "matplotlib>=3.4"
+        "matplotlib>=3.4, <3.10"
```

Comment:
Can you remind me why is it useful?

Reply:
This was a commit  by "actions-user", I have no clue why it is done.

[8df0035](https://github.com/gammapy/gammapy/pull/5451/commits/8df00357b2cc673d013d85d30cb7b9cff6a375b2)